packages/syntax-core/src/LinkButton/LinkButton.tsx

```diff
@@ -0,0 +1,126 @@
+import classNames from "classnames";
+import backgroundColor from "../colors/backgroundColor";
+import foregroundColor, {
+  foregroundTypographyColor,
+} from "../colors/foregroundColor";
+import React from "react";
+import { type Color, type Size } from "../constants";
+import Typography from "../Typography/Typography";
+
+import buttonStyles from "../Button/Button.module.css";
+import { textVariant, iconSize } from "../Button/ButtonConstants";
+
+import styles from "./LinkButton.module.css";
+
+/**
+ * LinkButton is a "variation" of Button that should look identical to Button, but should be used to render links instead.
+ * It needs to be wrapped with the appropriate wrapper for react-router, next/link, or a tag.
+ * The wrapper will most likely need a style={{textDecoration: "none"}} to remove the default underline styling.
+ *
+ * ```
+ * <a href="/english/resources" target="_blank" style={{textDecoration: "none"}}>
+ *    <LinkButton text="Resources" />
+ * </a>
+ * ```
+ *
+ * ```
+ * import Link from "next/link";
+ *
+ * <Link href="/english/resources" style={{textDecoration: "none"}}>
+ *    <LinkButton text="Resources" />
+ * </Link>
+ * ```
+ *
+ * ```
+ * import Link from "react-router-dom";
+ *
+ * <Link href="/english/resources" style={{textDecoration: "none"}}>
+ *    <LinkButton text="Resources" />
+ * </Link>
+ * ```
+ */
+export default function LinkButton({
+  text,
+  "data-testid": dataTestId,
+  color = "primary",
+  size = "md",
+  fullWidth = false,
+  startIcon: StartIcon,
+  endIcon: EndIcon,
+}: {
+  /**
```

Comment:
@somethiiing could we add `target` and `href` attributes now that we're going to return an anchor tag: `<a />`